concreate/schema/common.yaml

```diff
@@ -72,3 +72,6 @@ map:
         map:
           name: {type: str}
           branch: {type: str}
+  volumes:
+    seq:
+      - {type: str}
```

Comment:
if we do it this way, volumes cannot be overridden, only appended, do we really want this?

Reply:
So, this was something I was thinking about and I went with this approach for a simple reason: this is the way it is described in docs and tests, but it was just missing in the schema. I could update it to sequence of map though.

Another possibility is to actually merge two lists: `list1 + list2` in our merging algorithm.

Reply:
how to merge? we are doing list1 + list2, issue is if you want to remove/change volume. Because with plain list we will very soon get bug that we cannt override them similar to repositories.

Reply:
Oh, yes, sorry, I was too quick.

So, what to do with this? I'm actually cool with either option.